clientcontroller/opstackl2/consumer.go

```diff
@@ -353,25 +354,26 @@ func (cc *OPStackL2ConsumerController) QueryIsBlockFinalized(height uint64) (boo
 }
 
 // QueryActivatedHeight returns the L2 block number at which the finality gadget is activated.
-// It is fetched from the configuration of a CosmWasm contract OP finality gadget.
 func (cc *OPStackL2ConsumerController) QueryActivatedHeight() (uint64, error) {
-	queryMsg := &QueryMsg{Config: &Config{}}
-	jsonData, err := json.Marshal(queryMsg) // `{"config":{}}`
+	finalityGadgetClient, err := fgclient.NewFinalityGadgetGrpcClient(cc.Cfg.BabylonFinalityGadgetRpc)
 	if err != nil {
-		return 0, fmt.Errorf("failed marshaling to JSON: %w", err)
+		cc.logger.Error("failed to initialize Babylon Finality Gadget Grpc client", zap.Error(err))
+		return 0, err
```

Comment:
not sure why we put 0 there in the past. but I feel we should return max of uint64 on error. wdyt? same for the rest place as well as in `GetBlockNumberByTimestamp`

Reply:
yes, max of uint64 makes sense.